modules/conversion-center/widgets/link-in-bio.php

```diff
@@ -0,0 +1,1200 @@
+									'Vimeo'       => esc_html__( 'Vimeo', 'elementor' ),
+								],
+							],
+						],
+						'default' => 'Facebook',
+					],
+					[
+						'name'        => 'icon_icon',
+						'label'       => esc_html__( 'Content', 'elementor' ),
+						'type'        => Controls_Manager::ICONS,
+						'default'     => [
+							'url' => \Elementor\Utils::get_placeholder_image_src(),
+						],
+						'label_block' => true,
+					],
+					[
+						'name'         => 'icon_url',
+						'label'        => esc_html__( 'Link', 'elementor' ),
+						'type'         => Controls_Manager::URL,
+						'options'      => false,
+						'dynamic'      => [
+							'active' => true,
+						],
+						'autocomplete' => true,
+						'label_block'  => true,
+						'placeholder'  => esc_html__( 'Enter your link', 'elementor' ),
+						'condition'    => [
+							'icon_platform' => [
+								'Vimeo',
+								'Facebook',
+								'SoundCloud',
+								'Spotify',
+								'Instagram',
+								'LinkedIn',
+								'Pinterest',
+								'TikTok',
+								'X (Twitter)',
+								'YouTube',
+								'Apple Music',
+								'Behance',
+								'Dribble',
+								'Spotify',
+								'SoundCloud',
+								'Vimeo',
+								'Waze',
+							],
+						],
+					],
+					[
+						'name'        => 'icon_number',
+						'label'       => esc_html__( 'Number', 'elementor' ),
+						'type'        => Controls_Manager::TEXT,
+						'dynamic'     => [
+							'active' => true,
+						],
+						'label_block' => true,
+						'placeholder' => esc_html__( 'Enter your number', 'elementor' ),
+						'condition'   => [
+							'icon_platform' => [
+								'Telephone',
+								'Telegram',
+								'WhatsApp',
+							],
+						],
+					],
+					[
+						'name'        => 'icon_mail',
+						'label'       => esc_html__( 'Mail', 'elementor' ),
+						'type'        => Controls_Manager::TEXT,
+						'placeholder' => esc_html__( 'Enter your email', 'elementor' ),
+						'dynamic'     => [
+							'active' => true,
+						],
+						'label_block' => true,
+						'condition'   => [
+							'icon_platform' => [
+								'Email',
+							],
+						],
+					],
+				],
+				'title_field'   => "<i class='{{{ icon_icon.value }}}' ></i> {{{ icon_platform }}}",
+				'prevent_empty' => true,
+				'button_text'   => esc_html__( 'Add Icon', 'elementor' ),
+			]
+		);
+
+		$this->end_controls_section();
+	}
+
+
+	private function add_style_tab(): void {
+
+		$this->start_controls_section(
+			'identity_section_style',
+			[
+				'label' => esc_html__( 'Identity', 'elementor' ),
+				'tab'   => Controls_Manager::TAB_STYLE,
+			]
+		);
+
+		$this->add_control(
+			'identity_image_size',
+			[
+				'label'          => esc_html__( 'Image Size', 'elementor' ),
+				'type'           => Controls_Manager::SLIDER,
+				'default'        => [
+					'size' => 30,
+					'unit' => '%',
+				],
+				'tablet_default' => [
+					'unit' => '%',
+				],
+				'mobile_default' => [
+					'unit' => '%',
+				],
```

Comment:
Is it really necessary?

Reply:
If it is, what about the other breakpoints?